<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>produitmain</class>
 <widget class="QDialog" name="produitmain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1161</width>
     <height>721</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestion des produits</string>
   </property>
   <widget class="QTabWidget" name="tabWidget_crud">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>1101</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="afficherp">
     <attribute name="title">
      <string>Affficher</string>
     </attribute>
     <widget class="QTableView" name="TableAfficher">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="Tri_ident">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>270</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>TRI A IDENT</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Tri_ident_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>TRI D IDENT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>290</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_pdf">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>370</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Export PDF </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_supp">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>440</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supp_id">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_modifier">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>330</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_type_modifier">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>370</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_qte_modifier">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>410</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>330</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>370</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>410</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Qte</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_modifier_2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>460</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>30</y>
        <width>321</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>QR CODE :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>430</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ByteReceived</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ajouterp">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_ajouter">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_type_ajouter">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Qte</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id_ajouter">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_qte_ajouter">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>130</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ConnectedThings</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_ard">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>500</width>
        <height>500</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_simulation">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>150</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Simulation</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_byte_received">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>360</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ard">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>450</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_blue">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>210</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable Bluetooth Action</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>270</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>430</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Last State</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_retour">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
